app/views/items/search.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>Search Results</h1>
+<% if @search.count > 0 %>
+ <table WIDTH="50%" BORDER="2" ALIGN="CENTER" CELLSPACING="40">
+        <tr ALIGN="Center">
+            <td>Owner</td>
+            <td>Item Name</td>
+            <td>Price</td>
+            <td>Quantity</td>
+            <td>Description</td>
+            <td></td>
+        </tr>
+        <% @search.each do |item| %>
```

Comment:
Hmm. It bothers me that we repeat this chunk of code so often. I wonder if there's a more modular way of doing this?